website/blog/2024-04-22-0.74-yoga-improvements-preparing-for-the-new-architecture.md

```diff
@@ -0,0 +1,441 @@
+- [Android SDK minimum bump (Android 6.0)](/blog/2024/04/22/0.74-yoga-improvements-preparing-for-the-new-architecture#android-sdk-minimum-bump-android-60)
+- [Yarn 3 for New Projects](/blog/2024/04/22/0.74-yoga-improvements-preparing-for-the-new-architecture#yarn-3-for-new-projects)
+
+### Breaking Changes
+
+- [API Changes to PushNotificationIOS (Deprecated)](/blog/2024/04/22/0.74-yoga-improvements-preparing-for-the-new-architecture#api-changes-to-pushnotificationios-deprecated)
+- [Removal of deprecated `PropTypes`](/blog/2024/04/22/0.74-yoga-improvements-preparing-for-the-new-architecture#removal-of-deprecated-proptypes)
+- [Other Breaking Changes](/blog/2024/04/22/0.74-yoga-improvements-preparing-for-the-new-architecture#other-breaking-changes)
+
+<!--truncate-->
+
+## Highlights
+
+### Yoga 3.0
+
+#### New Layout Behaviors
+
+React Native 0.74 includes [Yoga 3.0](https://yogalayout.dev/blog/announcing-yoga-3.0), the newest version of our layout engine, which enables more correct layout behavior. These changes make styling more predictable, and enable rendering components written for the web with greater fidelity.
+
+React Native intentionally preserves some incorrect layout behaviors, where fixing them was found to effect a signigicant number of real-world components. Layout conformance will be able to be configured more granularly in future versions of React Native.
+
+:::warning
+
+React Native [previously flipped](https://yogalayout.dev/blog/announcing-yoga-3.0#correct-handling-of-logical-edges-in-row-reverse-containers) `left`/`right` (and `start`/`end`) edges when dealing with `margin`, `padding`, or `border`, set on a `row-reverse` container. Code which previously relied on edges being inverted may need to be updated to continue rendering correctly.
+
+<table>
+<tr>
+  <th>Style</th>
+  <th>Before</th>
+  <th>After</th>
+</tr>
+<tr>
+<td width="350">
+
+```jsx
+<View
+  style={{
+    flexDirection: 'row',
+    backgroundColor: 'red',
+    margin: 10,
+    width: 200,
+    height: 100,
+  }}>
+  <View
+    style={{
+      flexDirection: 'row-reverse',
+      backgroundColor: 'blue',
+      flex: 1,
+      marginLeft: 50,
+    }}>
+    <View
+      style={{
+        backgroundColor: 'green',
+        height: '50%',
+        flex: 1,
+        marginLeft: 50,
+      }}
+    />
+  </View>
+</View>
+```
+
+</td>
+<td>
+
+![Previous layout](../static/blog/assets/0.74-row-reverse-before.png)
+
+</td>
+<td>
+
+![New layout](../static/blog/assets/0.74-row-reverse-after.png)
+
+</td>
+</tr>
+</table>
+
+:::
+
+#### Support for `align-content: 'space-evenly'`
+
+Yoga 3.0 brings support for [`alignContent: 'space-evenly'`](https://developer.mozilla.org/en-US/docs/Web/CSS/align-content#space-evenly). This mode distributes the lines in a multi-line flex container using evenly spaced gaps, placed between line and container edges.
```

Comment:
afaik, "mode" is not a thing, it's a value for the property but you could clarify by restating the property name.

```suggestion
Yoga 3.0 brings support for [`alignContent: 'space-evenly'`](https://developer.mozilla.org/en-US/docs/Web/CSS/align-content#space-evenly). `space-evenly` distributes the lines in a multi-line flex container using evenly spaced gaps, placed between line and container edges.
```